pkg/parse/run.go

```diff
@@ -66,8 +67,17 @@ const (
 	RenderingNotRequired string = "Rendering not required but is currently enabled"
 )
 
+// RunOpts are the options used when calling Run
+type RunOpts struct {
+	runFn           RunFn
```

Comment:
IME, func type names usually end in -Func. That way variables and fields can use -Fn without colliding. In this case there's no collision because the type is public, but I'd still recommend using -Func for consistency.

Reply:
Done